charts/tfy-karpenter-config/values.yaml

```diff
@@ -184,8 +29,47 @@ karpenter:
     ## @param karpenter.defaultNodeTemplate.detailedMonitoring
     # Set this to true to enable EC2 detailed cloudwatch monitoring
     detailedMonitoring: false
-
-  ## Defautl GPU Node Template Config
+    ## @param karpenter.defaultNodeTemplate.extraSubnetTags Additional tags for the subnet.
+    extraSubnetTags: {}
+    ## @param karpenter.defaultNodeTemplate.extraSecurityGroupTags Additional tags for the security group.
+    extraSecurityGroupTags: {}
+
```

Comment:
Let's add an `enabled` option here as well as we have done for `defaultNodePool`